### ISSUE TITLE
[BUG] False positive on T001

### ISSUE THREAD

**Issue by christopherpickering** [Riverside-Healthcare/djLint#114]
```html
<form method="post" action="{% if p %}{{url_for('project_bp.edit_project', project_id=p.id) }}{% else %}{{ url_for('project_bp.new_project') }}" id="project-new">
```

triggers a false positive on T001.
